futures-test/src/task/executor.rs

```diff
@@ -0,0 +1,8 @@
+//! Implementations of [`Executor`][futures_core::task::Executor] with various
```

Comment:
Maybe rename this to `executor/mod.rs` instead.

Reply:
Yeah, I've been getting quite annoyed at the new file scheme breaking tab completion, so will switch this back.

Reply:
They changed it because the old system can lead to loads of `mod.rs` file tabs in an editor. However, that's IMO fine if these files are just lightweight files that just reexport the right things from their child modules. So, I continue to like `mod.rs` files 🙂